tests/fixtures/search_addon/manifest.json

```diff
@@ -0,0 +1,22 @@
+{
+  "manifest_version": 2,
+  "name": "MozSearch search engine",
+  "description": "Adds a search engine",
+  "version": "1.0",
+  "browser_specific_settings": {
+    "gecko": {
+      "strict_min_version": "55.0"
```

Comment:
Not sure it matters, but is this a Firefox version? If so, feels oddly low (circa [Aug 2017, even](https://www.mozilla.org/en-US/firefox/55.0/releasenotes/))